From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Wed, 23 Oct 2024 20:55:44 +0200
Subject: Modernize Makefile

---
 Makefile | 13 ++++++++-----
 1 file changed, 8 insertions(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index e914573..4b56fb7 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,8 @@
 .PHONY: clean-pyc clean-build docs clean
 
+all: dist
+
+
 help:
 	@echo "clean-build - remove build artifacts"
 	@echo "clean-pyc - remove Python file artifacts"
@@ -37,11 +40,11 @@ docs:
 	tox -e docs
 
 dist: clean
-	python setup.py sdist
-	python setup.py bdist_wheel
-	python setup.py bdist_egg
-	ls -l dist
+	python -m build .
 
 PIP ?= pip
-install: dist
+install:
+	$(PIP) install .
+
+install-dist:
 	$(PIP) install -I $(shell ls -1rt dist/*.whl | tail -n 1)
