From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Wed, 23 Oct 2024 21:23:00 +0200
Subject: astunparse cannot reproduce this code, need to look into it

---
 lib/astunparse/unparser2j.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/astunparse/unparser2j.py b/lib/astunparse/unparser2j.py
index 7cbc200..ce47840 100644
--- a/lib/astunparse/unparser2j.py
+++ b/lib/astunparse/unparser2j.py
@@ -67,7 +67,7 @@ class Unparser2J:
         elif isinstance(tree, ASTNode):
             cname = tree._class
             meth = getattr(self, "_"+cname, None)
-            self.write(f'{"{"}"_class":{self.spacer}"{cname}"')
+            self.write('{' + f'"_class":{self.spacer}"{cname}"')
             if meth:
                 meth(tree)
             else:
@@ -83,7 +83,7 @@ class Unparser2J:
             self.write(f'{"}"}')
         elif isinstance(tree, bool):
             self.write(f'{"{"}"_class":{self.spacer}"bool",')
-            self.fill(f' "value": "{repr(tree)}"{"}"}')
+            self.fill(f' "value": "{repr(tree)}"' + '}')
         elif isinstance(tree, int):
             self.write(f'{tree:d}')
         elif isinstance(tree, float):
@@ -121,7 +121,7 @@ class Unparser2J:
                 self.write(f'{"{"}"_class":{self.spacer}"NoneType"{"}"}')
         else:
             cname = tree.__class__.__name__
-            self.write(f'{"{"}"_class":{self.spacer}"{cname}"')
+            self.write('{' + f'"_class":{self.spacer}"{cname}"')
             meth = getattr(self, "_"+cname, None)
             if meth:
                 meth(tree)
