From: Johannes Willkomm <johannes@johannes-willkomm.de>
Date: Wed, 23 Oct 2024 21:23:25 +0200
Subject: Test only local project files not entire library

---
 tests/common.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/common.py b/tests/common.py
index 0c14b13..cb8a1b9 100644
--- a/tests/common.py
+++ b/tests/common.py
@@ -176,8 +176,8 @@ class AstunparseCommonTestCase:
         raise NotImplementedError()
 
     test_directories = [
-        os.path.join(getattr(sys, 'real_prefix', sys.prefix),
-                     'lib', 'python%s.%s' % sys.version_info[:2])]
+        'lib/astunparse'
+    ]
 
     def test_files(self):
         names = []
